articles/multifactor-authentication/overview.md

```diff
@@ -0,0 +1,83 @@
+
+* **Knowledge**: Something the user **knows** (such as a password)
+* **Possession**: Something the user **has** (such as a mobile device)
+* **Inheritance**: Something the user **is** (such as a fingerprint or retina scan)
+
+## Implement MFA with Auth0
+
+Enabling MFA for your tenant is a fairly straightforward process. First, you toggle on the factors you choose to enable on your tenant, such as push notifications or SMS. Next, you perform any further setup required to configure that factor, and last, you choose whether you wish to force MFA for all users or not. See the instructions below for details.
+
+You can also [customize your MFA flow](/multifactor-authentication/custom) with [Auth0 Rules](/rules), to allow MFA to only be required in specific circumstances or force a particular factor to be used.
+
+### 1. Enable the factors you require
+
+In the [Dashboard > Multifactor Auth](${manage_url}/#/mfa), head to the Multifactor Auth section. Here you will find a series of toggles for the MFA factors supported by Auth0. 
+
+![MFA Dashboard Page](/media/articles/multifactor-authentication/mfa-dashboard-1.png)
+
+Any or all of these factors can be enabled simultaneously. When logging in the first time, the user will be shown a default, but will be allowed to choose another factor to use, if you have more than one factor enabled in the Dashboard. 
+
+::: note
+When you enable the SMS or the Duo factor, you will have to click on it and fill in a few further settings related specifically to that factor before continuing.
+:::
+
+#### Always require multi-factor authentication
+
+![MFA Dashboard Page](/media/articles/multifactor-authentication/mfa-dashboard-2.png)
+
+The **Always require Multi-factor Authentication** setting, when enabled, will allow authentication to your applications only when multi-factor authentication is used. This means that one or more of the factors is enabled in the Dashboard, and successfully completed by end-users when authenticating.
+
+### 2. Set up your services
+
+Auth0 supports the following factors for implementing MFA. You must enable at least one to use MFA, but you can choose to enable and make available more than one factor if you wish.
+
+* [One Time Passwords (OTP)](factors/otp)
+* [Push](factors/push)
+* [SMS](factors/sms)
+* [Duo](factors/duo)
+
+## Customizing multi-factor authentication
+
+### Customizing via Rules
+
+If you need to customize the multi-factor experience you are offering to your users, you may do so via [custom rules configurations for multi-factor authentication](/multifactor-authentication/custom). This might be needed, for example, if you wish to trigger MFA for only specific applications, or for specific users based on user metadata or on IP addresses.
+
+### Customizing Guardian
```

Comment:
I think we can say 'Customizing MFA', and mention this does not apply to duo. With the new dashboard, google-authenticator UI will only be shown for legacy rules, and using MFA instead of guardian here will help remove the association with the guardian app.